### PR TITLE
Delete select command

### PR DESCRIPTION
**Pull request by jsuresh**

### CODE REVIEW

File: src/Commands.hs [hunk 1]
```diff
@@ -47,57 +47,6 @@ import Types(IOR, REnv(..), getToolConfig, scopeInfo)
...
 
--- Make the specified release the live release, replacing any existing release.
-select :: T.Text -> IOR ()
-select release = do
-  tcfg <- getToolConfig
-  case tc_deployMode tcfg of
-    DeployMode_select -> selectNoProxy release
-    _ -> P.select release
-
-selectNoProxy :: T.Text -> IOR ()
```
Comment (timbod7) [inline src/Commands.hs:58]: I thought we were just going to get rid of the P.select functionality.  The selectNoProxy capability is useful when you want to start a single once in the bootscript for a machine and hence no frontend proxy is required.

Comment (timbod7) [reply src/Commands.hs:58]: Looking at your renamings (which look great!), I think that` hx-deploy-tool start` should run the existing "selectNoProxy" code when a proxy is not configured.

**Comment by jsuresh** [comment]
builds on https://github.com/helix-collective/hx-deploy-tool/pull/22

**Comment by jsuresh** [comment]
@timbod7 I forgot what we discussed here? Was the plan to just drop this PR, and instead delete the entire 'no proxy' branch for now?

**Comment by timbod7** [comment]
I think the plan was to:

- remove the select cli command
- remove `ProxyMode.select`
- rename `selectNoProxy` to `startNoProxy`, and call it from the cli start command.

**Pull request closed without merging**